skill/templates/artifact-format.md

```
@@ -0,0 +1,134 @@
+
+# Record it
+kata artifact record "$RUN_ID" \
+  --stage build \
+  --flavor rust-compilation \
+  --step compile \
+  --file /tmp/build-report.md \
+  --summary "Cargo build --release output: 0 errors, 2 warnings (unused import in auth.rs)"
+```
+
+**`--json` output**:
+```json
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)